tests/integration/api/test_ruleset.py

```diff
@@ -0,0 +1,129 @@
+import asyncio
+from unittest.mock import patch
+
+import aiodocker
+
+from eda_server import ruleset
+from eda_server.config.enums import DeploymentTypeEnum
+
+activation_id = 1
+large_data_id = 1
+execution_environment = "fedora:36"
+rulesets = ""
+ruleset_sources = []
+inventory = ""
+extravars = ""
+working_directory = "/tmp"
+host = "test-eda-server"
+port = 15000
+
+
+class TstDocker(aiodocker.docker.Docker):
+    def __init__(self, *args, **kwargs):
+        pass
+
+
+class TstDockerContainer(aiodocker.docker.DockerContainer):
+    def __init__(self, *args, **kwargs):
+        pass
+
+    def __getitem__(self, key):
+        return self
+
+
+class TstProc(asyncio.subprocess.Process):
+    def __init__(self, *args, **kwargs):
+        self.pid = 19999
+
+
+def test_resolve_activation_function():
+    expected = "create_websocket_docker_activation"
+    src = {}
+    f = ruleset.resolve_activation_function(src)
+    assert f.__name__ == expected
```

Comment:
You can check identity against the function itself, not it's name

```
f is create_websocket_docker_activation
```

Reply:
Yeah. I can fix that.